aces/joint_deconvolution/make_region_list.py

```diff
@@ -0,0 +1,69 @@
+# flake8: noqa
```

Comment:
You should enable flake8 qa.  It'll tell you to change a few whitespace things, but it's better to have the syntax checking enabled

Reply:
I've removed this line from both this and #291, but the tests seem to have failed. Is this just because my code isn't consistent with the expected style?

Reply:
Yes.  See the error message:
https://github.com/ACES-CMZ/reduction_ACES/actions/runs/3428414763/jobs/5712661488#step:7:13

You can put # noqa or whatever the directive is at the end of lines you don't want to change, or we can even disable some of the checks (I think it's OK to have extra whitespace around `=`, for example).  Let me know what you prefer

Reply:
yeah that's fine - just find a way to mark only those lines as #noqa, rather than the whole script (there's a way to tell it to just ignore specific warnings)